into_repo.sh

```diff
@@ -0,0 +1,3 @@
+# override xenctrl.system to obtain an opam repository without internal xenserver dependencies
+curl -L https://raw.githubusercontent.com/xapi-project/ocaml-xen-lowlevel-libs/master/opam > packages/xs/xenctrl.system/opam
+echo 'archive: "https://github.com/xapi-project/ocaml-xen-lowlevel-libs/archive/master.tar.gz"' > packages/xs/xenctrl.system/url
```

Comment:
Unfortunately that is a dependency of some of the packages in `xs` and needs to stay there